scripts/vendor-boringssl.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/sh
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftNIO open source project
+##
+## Copyright (c) 2018 Apple Inc. and the SwiftNIO project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+#
+# Copyright 2016, gRPC Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script creates a vendored copy of BoringSSL that is
+# suitable for building with the Swift Package Manager.
+#
+# Usage: 
+#   1. Run this script in the package root. It will place 
+#      a local copy of the BoringSSL sources in Sources/CNIOBoringSSL.
+#      Any prior contents of Sources/CNIOBoringSSL will be deleted.
+#
+set -eou pipefail
+
+HERE=$(pwd)
+DSTROOT=Sources/CNIOBoringSSL
+TMPDIR=$(mktemp -d /tmp/.workingXXXXXX)
+SRCROOT="${TMPDIR}/src/boringssl.googlesource.com/boringssl"
+
+echo "REMOVING any previously-vendored BoringSSL code"
+rm -rf $DSTROOT/include
```

Comment:
quotes missing, can we run this through `shellcheck`?  Especially the lines starting with `rm -rf`